AK/MACAddress.h

```diff
@@ -18,6 +18,12 @@
 #    include <AK/ByteString.h>
 #endif
 
+namespace AK {
+
+class MACAddress;
+template<>
+struct Traits<MACAddress>;
```

Comment:
These forward declarations seem unrelated.
I think you meant to add them in the next commit?